keras-dreambooth-sprint/README.md

```diff
@@ -1,95 +1,135 @@
+[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing). If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras).
 
-unique_id = "sks"
-class_label = "dog"
-prompt = f"A photo of {unique_id} {class_label} in a bucket"
-image = pipeline(prompt, num_inference_steps=50).images[0]
-```
+Note/q: having a colab and then a repo could maybe be confusing?
+Note: maybe let's change the repo_id in push_to_hub_keras to push to the organization.
 
-### **Model Hosting**
+## Model Hosting
 
-At the end of this notebook you will see a section dedicated for hosting, and a separate one for inference. We will be using huggingface_hub library’s Keras-specific model pushing and loading functions, `push_to_hub_keras` and `from_pretrained_keras` . We will first push the model using `push_to_hub_keras`. After model is pushed, you will see the model is hosted with a model card like below 👇 
+At the end of this notebook you will see a section dedicated for hosting and a separate one for inference. We will be using the `huggingface_hub` library’s Keras-specific model pushing and loading functions: `push_to_hub_keras` and `from_pretrained_keras` . We will first push the model using `push_to_hub_keras`. After the model is pushed, you will see the model is hosted with a model card like below 👇 
 
 ![Repository](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/dreamboothrepo.png)
 
-to version the models better, enable discoverability and reproducibility, we will fill the model card.  Click `Edit model card` . We will first fill the Metadata section of the model card. If your model is trained with a dataset from Hugging Face Hub, you can fill the datasets section with the dataset. We will provide fill `pipeline_tag` with `Text-to-Image` and pick a license for our model. 
+To version the models better, enable discoverability and reproducibility, we will fill the model card. Click `Edit model card` . We will first fill the Metadata section of the model card. If your model is trained with a dataset from the Hugging Face Hub, you can fill the datasets section with the dataset. We will provide fill `pipeline_tag` with `Ttext-to-image` and pick a license for our model. 
 
 ![Metadata](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/dreambooth-etadata.png)
 
-Then, we will fill the markdown part. Hyperparameters and plot is automatically generated so we can write a short explanation for description, intended use and dataset.
-
+Then, we will fill the markdown part. Hyperparameters and plot are automatically generated so we can write a short explanation for description, intended use and dataset.
 
 You can find the example repository below 👇 
 
 [keras-dreambooth/dreambooth_diffusion_model · Hugging Face](https://huggingface.co/keras-dreambooth/dreambooth_diffusion_model)
```

Comment:
I think this name needs to change to `keras-dreambooth/dreambooth_diffusion_model`?